006-admin-server.md

```diff
@@ -0,0 +1,116 @@
+# Strimzi Admin Server
+
+The Strimzi Admin repository was setup in December 2019 to hold an implementation of an Admin API. The repository has not had any contributions yet so this proposal sets out what the structure of an Admin Server might look like in order to get to the initial implementation of the server.
+
+## Motivation
+
+An API server will:
+* provide a consolidated backend API for additional interfaces like a browser based UI or a CLI.
+* be capable of supporting both REST and GraphQL interfaces.
+* allow for more sophisticated APIs to be built on top of the existing APIs like the kubernetes API providing value-add to existing content.
+
+## Proposal
+
+### The server
+The proposal is to make a modular server using the Vert.x-Web toolkit which has strong support for creating both REST and GraphQL interfaces.
+The server contains a single Vert.x verticle which listens for inbound traffic on 1 or more configurable ports.
+Allowing multiple configurable ports will allow the server to expose different functionality on each port, the simplest example of this being exposing an HTTP API on one port and a separate HTTPS API on a different port.
```

Comment:
Maybe you can add some examples for clarification what exactly does this mean and what could be the use-case?

Reply:
I have provided the example of supporting multiple authentication schemes to demonstrate the use case. The idea behind this is to provide something similar to the `advertised listeners` in Kafka.